transforms/deprecate-create/deprecate-create.js

```diff
@@ -0,0 +1,143 @@
+/*
+ * Convert all <Create /> to
+ * <Button buttonType=“dashed” iconType=“plus” iconPosition=“after” fullWidth />
+ */
+import registerMethods from "../registerMethods";
+
+function transformer(fileInfo, api, options) {
+  const j = api.jscodeshift;
+  registerMethods(j);
+
+  const newProps = [
```

Comment:
If we use my suggestion below, these props would be moved into the method as they're only used in one place.

Reply:
Done